cmd/deploytarget.go

```diff
@@ -454,13 +459,13 @@ func init() {
 	addDeployTargetCmd.Flags().StringP("build-image", "", "", "DeployTarget build image to use (if different to the default)")
 
 	addDeployTargetToOrganizationCmd.Flags().StringP("name", "O", "", "Name of Organization")
-	addDeployTargetToOrganizationCmd.Flags().UintP("deploy-target", "D", 0, "ID of DeployTarget")
+	addDeployTargetToOrganizationCmd.Flags().UintP("deploytarget", "D", 0, "ID of DeployTarget")
 
 	deleteDeployTargetCmd.Flags().UintP("id", "", 0, "ID of the DeployTarget")
 	deleteDeployTargetCmd.Flags().StringP("name", "", "", "Name of DeployTarget")
 
-	RemoveDeployTargetFromOrganizationCmd.Flags().StringP("name", "O", "", "Name of Organization")
-	RemoveDeployTargetFromOrganizationCmd.Flags().UintP("deploy-target", "D", 0, "ID of DeployTarget")
+	removeDeployTargetFromOrganizationCmd.Flags().StringP("name", "O", "", "Name of Organization")
```

Comment:
Can we change this to `organization` to match other flags in other commands. There may be other organization commands that use name, but if they can be updated to be `organization` that'd be wicked. That way there is no confusion across the flagset for organizations.

It was initially confusing to me when i went to use `--name` in one place (`lagoon list organization-users --organization ${orgname}`) only to find it was `--organization` and then vice versa when using a different command (`lagoon delete organization-deploytarget --name ${orgname}`).

Reply:
I think the only one that should be `--name` is maybe `add organization`.

Alternatively, if the flag is changed to `organization-name` then we cover all possibly cases on all commands

Reply:
Sounds good, I've changed the flag to `organization-name`.